apps/onboarding/src/session/session.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { getRepositoryToken } from '@nestjs/typeorm'
+import{ Session, SessionRepositoryFake } from './session.entity'
+import { SessionService } from './session.service'
+
+
+describe('SessionService', () => {
+  let service: SessionService
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule(
+   {
+      providers: [SessionService,
+        {
+          provide: getRepositoryToken(Session),
+          useValue: SessionRepositoryFake,
+        },
+      ]
+  }).compile()
+
+    service = module.get<SessionService>(SessionService)
+  })
+
+  it('should be defined', () => {
+    expect(service).toBeDefined()
+  })
+
+  it('should create a session object', async () => {
+    const session = Session.of({externalAccount:'test', id: '1'})
```

Comment:
Looks like you're mocking the service so only testing the mocking itself.
You should mock the repository instead and test the service.